addons/binding/org.openhab.binding.rfxcom/README.md

```diff
@@ -104,6 +105,8 @@ This binding currently supports following channels:
 | motion | Switch | Motion detection sensor state. |
 | rainrate | Number | Rain fall rate in millimeters per hour. |
 | raintotal | Number | Total rain in millimeters. |
+| rawmessage | String | Hexadecimal string of the raw RF message. |
```

Comment:
I mean that all bindings could have a rawmessage and rawpayload channel if we wanted, so we could expose the raw bytes to rules if they want them. Would be another PR if it got done anyway.